src/main/java/pokecube/legends/init/ItemInit.java

```diff
@@ -74,7 +74,7 @@
     
     //Raids/Dynamax/Gigantamax
     public static final RegistryObject<Item> WISHING_PIECE;
-    public static final RegistryObject<Item> GIGANTIC_SHARD;
+    //public static final RegistryObject<Item> GIGANTIC_SHARD;
```

Comment:
Do not remove the item after the update was already released to add it, this will cause incompatiblity issues... just mark in the name that it isn't working